datatypes/src/raster/data_type.rs

```diff
@@ -150,3 +152,21 @@ where
         R::TYPE
     }
 }
+
+impl RasterDataType {
+    pub fn ocl_type(self) -> String {
```

Comment:
i think this is an "ocl_type_name"?

Reply:
91ef24f5b869f75563586b32b932afa0a27fce0f